codice/ddf
# DDF-1966 - added support to query by GMD_Metadata from CSW Source

#### What does this PR do?
Adds support to query by the GMD Metadata / CSW ISO profile from the CSW Source
#### Who is reviewing it?
@jlcsmith @bdeining @glenhein
#### How should this be tested?
Configure a GMD CSW Federated Source - defaults should be sufficient - you can use server:
http://csw.data.gov.uk/geonetwork/srv/en/csw
#### Any background context you want to provide?
#### What are the relevant tickets?
DDF-1966
#### Screenshots (if appropriate)
#### Checklist:

Conflicts:
	catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/GmdMetacardType.java
	catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java
	catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/CswFilterDelegate.java
	catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/blueprint/blueprint.xml
